xml/System/ArgumentOutOfRangeException.xml

```diff
@@ -47,160 +47,165 @@
   
- Typically, an <xref:System.ArgumentOutOfRangeException> results from developer error. Instead of handling the exception in a `try`/`catch` block, you should eliminate the cause of the exception or, if the argument is returned by a method call or input by the user before being passed to the method that throws the exception, you should validate arguments before passing them to the method.  
+Typically, an <xref:System.ArgumentOutOfRangeException> results from developer error. Instead of handling the exception in a `try`/`catch` block, you should eliminate the cause of the exception or, if the argument is returned by a method call or input by the user before being passed to the method that throws the exception, you should validate arguments before passing them to the method.  
   
- <xref:System.ArgumentOutOfRangeException> is used extensively by:  
+<xref:System.ArgumentOutOfRangeException> is used extensively by:  
   
 -   Classes in the <xref:System.Collections> and <xref:System.IO> namespaces.  
   
 -   The <xref:System.Array> class.  
   
 -   String manipulation methods in the <xref:System.String> class.  
   
- The conditions in which an <xref:System.ArgumentOutOfRangeException> exception is thrown include the following:  
-  
- You are retrieving the member of a collection by its index number, and the index number is invalid.  
- This is the most common cause of an <xref:System.ArgumentOutOfRangeException> exception.   Typically, the index number is invalid for one of three reasons:  
-  
--   The collection has no members, and your code assumes that it does. The following example attempts to retrieve the first element of a collection that has no elements:  
-  
-     [!code-csharp[System.ArgumentOutOfRangeException#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/cs/NoElements.cs#4)]
-     [!code-vb[System.ArgumentOutOfRangeException#4](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/vb/NoElements.vb#4)]  
-  
-     To prevent the exception, check whether the collection's `Count` property is greater than zero before attempting to retrieve any members, as the following code fragment does.  
+The conditions in which an <xref:System.ArgumentOutOfRangeException> exception is thrown include the following:  
   
-     [!code-csharp[System.ArgumentOutOfRangeException#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/cs/NoElements.cs#5)]
-     [!code-vb[System.ArgumentOutOfRangeException#5](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/System.ArgumentOutOfRangeException/vb/NoElements.vb#5)]  
+- You are retrieving the member of a collection by its index number, and the index number is invalid.  
+
+  This is the most common cause of an <xref:System.ArgumentOutOfRangeException> exception.   Typically, the index number is invalid for one of three reasons:  
```

Comment:
We say there are three reasons but we have four bullets. Also, should we use numbered list for these reasons?